README.rst

```diff
@@ -2,7 +2,13 @@
 AOSOM-Streaming
 ###############
 
+**************
 Version v0.6.1
```

Comment:
Previously this line was normal text - now it's a heading. Is this intentional?

Reply:
Yes